fcli-core/fcli-sc-sast/src/main/java/com/fortify/cli/sc_sast/_common/session/cli/cmd/SCSastSessionLoginCommand.java

```diff
@@ -41,7 +41,7 @@ protected void logoutBeforeNewLogin(String sessionName, SCSastSessionDescriptor
     
     @Override
     protected SCSastSessionDescriptor login(String sessionName) {
-        IUrlConfig urlConfig = sessionLoginOptions.getUrlConfigOptions();
+        ISCSastAndSSCUrlConfig urlConfig = sessionLoginOptions.getUrlConfigOptions();
```

Comment:
For consistency/clarity, please change variable name to something like `scSastAndSscUrlConfig`